src/UnitTests/Arch/Intel/DeadConditionFlagsFinderTests.cs

```diff
@@ -33,6 +33,7 @@ public class DeadConditionFlagsFinderTests
 	{
 		private DeadConditionFlagsFinder dcff;
 
+//		[OneTimeSetup]
```

Comment:
We try to avoid commenting out code without specifying why it is commented out. For all of these tags, consider adding the following line (editing for correctness)
```
// Once the project has been ported to use NUnit 3.0, remove this comment and uncomment the line below.
```
Do thus for all occurrences of `[OneTimeSetup]`.